tests/test_decorators.py

```diff
@@ -30,6 +31,8 @@
 
 api = hug.API(__name__)
 module = sys.modules[__name__]
+test_directory = os.path.dirname(os.path.realpath(__file__))
```

Comment:
No but we could create one for the testing directory if need be, either just using `__init__.py` or a `constants.py` file or something. Honestly, my preferred solution to this issue is just to setup an alias for running the tests from desired root directory, as done here: https://github.com/timothycrosley/hug/blob/develop/.env#L73 if you use the autoenv environment. The reason I prefer this approach, is that putting the logic in the code only solves a very small number of use cases (for instance: if you run py.test from the root of the project blankly and you have a typical virtual env setup, it will tests unrelated things making the output useless) However, I'm cool  putting the intelligence into the test code itself, as long as we only have to declare once.